services/QuillLMS/app/assets/stylesheets/pages/blog_posts.scss

```diff
@@ -47,6 +50,12 @@ $blogpostlightteal: rgba(44, 127, 155, 0.08);
     .quill-button {
       background-color: $blogpostgreen;
     }
+    .quill-button:hover, .quill-button:focus {
+      background-color: $blogpostgreen;
+    }
+    .quill-button:focus {
+      outline: dashed 2px $blogpostgreen;
```

Comment:
I don't have an issue with this, but I think it's worth noting that this is a departure from what we've done historically with the focus outlines (which need to contrast the background, but otherwise don't care what color the item is that they're focusing on).